go.mod

```diff
@@ -94,10 +94,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
 
-replace (
-	github.com/opendatahub-io/opendatahub-operator/v2/cmd/tools => ./cmd/tools
-	k8s.io/api => k8s.io/api v0.28.4
-	k8s.io/client-go => k8s.io/client-go v0.28.4
-)
+replace github.com/opendatahub-io/opendatahub-operator/v2/cmd/tools => ./cmd/tools
```

Comment:
why we have this one at first? :thinking:

Reply:
@zdtsw It looks like it was introduced [here](https://github.com/opendatahub-io/opendatahub-operator/pull/1189/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R100-R101) to make the dependencies happy for a controller-runtime version bump.

Reply:
hum, i mean github.com/opendatahub-io/opendatahub-operator/v2/cmd/tools, this does not seems like needed

Reply:
@zdtsw My mistake thank you for clarifying. It's removed from `go.mod` in the latest commit.